### PR TITLE
Merge MONDO:0004031 ovary mixed epithelial carcinoma into MONDO:0004032 ovarian seromucinous carcinoma

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/mondo/issues/7609

### CODE REVIEW

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:74355]: @sabrinatoro [NCIT:40090](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes?short_form=NCIT_C40090) no longer has any of the synonyms that are included here with NCIT:40090 as the source. Should those also be edited in this ticket? The [Physician Data Query ontology](https://bioportal.bioontology.org/ontologies/PDQ/?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FPDQ%2FCDR0000039597) in BioPortal does include these synonyms and is mapped in NCIT:40090 so PDQ:CDR0000039597 could be the source.

File: src/ontology/mondo-edit.obo
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:74368]: duplicated with above. I think a normalization step will do the trick

File: src/ontology/mondo-edit.obo
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:74355]: @twhetzel yes, let's update the synonyms since we are here. I can do that.

File: src/ontology/mondo-edit.obo
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:74368]: sure, are you leaving that for me to update?

**Review by sabrinatoro — APPROVED** [review]
Approving. Will merge after the QC checks have passed.

**Comment by sabrinatoro** [comment]
I merged faster than expected. I will follow up on this and make sure all is well before merging any other PR

**Pull request merged**
